app/celery/tasks.py

```diff
@@ -886,13 +884,13 @@ def send_notify_no_reply(self, data):
             )
         ]
         current_app.logger.info(f"Data we are sending to persist_notifications is {data_to_send}")
-        saved_notification = persist_notifications(data_to_send)
-        send_notification_to_queue(saved_notification, False, queue=QueueNames.NOTIFY)
+        saved_notifications = persist_notifications(data_to_send)
+        if saved_notifications and len(saved_notifications) == 1:
+            send_notification_to_queue(saved_notifications[0], False, queue=QueueNames.NOTIFY)
```

Comment:
What happens if the length is not 1? Do we discard these? Can't we safely assume it will always be one?

Reply:
if the length is not 1, it means we didn't save. I don't know what we should do here? I think this check should always pass but I can remove it if that makes more sense.

Reply:
Yeah I'd remove, it would make sense. If it is not 1, it will crash and we will know. If it is not 1, we will never know and won't be able to bring a fix.

Reply:
done